packages/harpguru-core/src/hooks/use-is-zoomed-columnbounds/use-is-zoomed-columnbounds.ts

```diff
@@ -0,0 +1,21 @@
+import { useGlobal } from 'reactn'
+
+import { doSparceIdedObjectMatricesMatch } from '../../packages/do-sparce-ided-object-matrices-match'
+
+// TODO: Add a test
```

Comment:
Test required, but I think this should be raised as techdebt.

Reply:
We're not likely to change this function in the future.

Reply:
Raised as techdebt in https://github.com/js-jslog/harpguru/issues/128#issuecomment-967035347